c_glib/gandiva-glib/native-function.h

```diff
@@ -51,7 +51,8 @@ struct _GGandivaNativeFunctionClass
   GObjectClass parent_class;
 };
 
-GGandivaFunctionSignature *ggandiva_native_function_get_signature(GGandivaNativeFunction *native_function);
+GList *
+ggandiva_native_function_get_signatures(GGandivaNativeFunction *native_function);
```

Comment:
Do we add `GARROW_AVAILABLE_IN_1_0` in another pull request after #4841 is merged?

Reply:
Yes.

Reply:
OK.